README_LINUX.md

```diff
@@ -213,8 +212,8 @@ This assumes your ccache executables are installed into `/usr/lib/ccache` - you
 Building a Debian package
 -------------------------
 
-The most up-to-date debian packaging rules are maintained by the
-Debian Multimedia team. Repository (with debian/ folder):
+The most up-to-date Debian packaging rules are maintained by the
+debian Multimedia team. Repository (with debian/ folder):
```

Comment:
capitalize Debian in "debian Multimedia"

Reply:
I'm also going to suggest -- this is a good example demonstrating why it's important to review your own changes carefully *before* committing.

In original line 216, you changed lowercase debian to capitalized Debian -- correct.

Then, in the next line, you changed a capitalized Debian to lowercase -- huh? By what rule? *It was correct before!* Why did you change it?

It gives the appearance of revisions being done very fast, without paying attention to detail. If you want to help the project, this is not a good way to do it.

Reply:
I think it was a confusion because [this comment](https://github.com/supercollider/supercollider/pull/3668#discussion_r181218478) wasn't clear enough for a non-native speaker